modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,296 @@
 [[custom-ssl]]
-= Using a Custom SSL Certificate
+= Custom SSL Certificates
 
+You can use custom certificates with {productname} and {productname} Proxy.
 
+This section covers how to use a third party SSL certificate authority with a new {productname} installation, and replacing existing certificate with new custom certificates.
 
+Before you begin, ensure you have:
 
+* A certificate authority (CA) SSL public certificate
+* An SSL server key
+* An SSL server certificate
 
+Your key and certificate files must be in PEM format.
 
-The following section will guide you through using a custom certificate with {productname} {productnumber} and {susemgrproxy} {productnumber}.
+The host name of the SSL keys and certificates must match the fully qualified host name of the machine you deploy them on.
+You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
+You can also list multiple host names if your environment requires it.
 
+If you want to use intermediate certificates, you need to merge the intermediate and root CA certificates into one file.
+Ensure that the intermediate certificate comes first in the combined file.
 
 
+== Custom Certificates for New Installations
 
-[[bp.cert.custom.req]]
-== Prerequisites
+By default, {productname} uses a self-signed certificate.
+After you have completed the initial setup, you can replace the default certificate with a custom certificate.
 
+.Procedure: Installing Custom Certificates on a New {productname} Server
 
-The following list provides requirements for using a custom certificate.
-
-* A Certificate Authority (CA) SSL public certificate file
-* A Web server SSL private key file
-* A Web server SSL public certificate file
-* Key and Certificate files must be in PEM format
-
+. Install the {productname} Server according to the instructions in xref:installation:install-intro.adoc[].
+. Complete the initial setup according to xref:installation:server-setup.adoc[].
+. At the command prompt, point the SSL environment variables to the custom certificate file locations:
++
+----
+export CA_CERT=`path_to_CA_certificate_file`
+export SERVER_KEY=`path_to_web_server_key`
+export SERVER_CERT=`path_to_web_server_certificate`
+----
```

Comment:
I've left the caps in, because the actual syntax requires the "quotes" and I didn't want to add more punctuation in. Replace the ``backticks`` with <pointy braces> though.